tests/test_integration.py

```diff
@@ -56,10 +57,11 @@ def test_single_task_benchmark_clf_loop_with_multiple_test_sets(
 
     y_prob = {}
     y_pred = {}
+    task_name = test_single_task_benchmark_clf_multiple_test_sets.target_cols[0]
     for k, test_subset in test.items():
         x_test = np.array([dm.to_fp(dm.to_mol(smi)) for smi in test_subset.inputs])
-        y_prob[k] = model.predict_proba(x_test)[:, :1]  # for binary classification
-        y_pred[k] = model.predict(x_test)
+        y_prob[k] = {task_name: model.predict_proba(x_test)[:, :1]}  # for binary classification
```

Comment:
**Question:** Are you required to specify the prediction as a dictionary here? I would think not from the code.

Reply:
Yes, the new `evaluate_benchmark` method does expect it to be a dictionary for single task, multi-set benchmarks.